### PR TITLE
gnome-obfuscate: 0.0.7 -> 0.0.9

### LINKED ISSUE #229506: Update request: gnome-obfuscate 0.0.7 → 0.0.9
- Package name: gnome-obfuscate
- Latest released version: 0.0.9

- Current version on the unstable channel: 0.0.7
- Current version on the stable/release channel: 0.0.7

- [yes] Checked the [nixpkgs pull requests](https://github.com/NixOS/nixpkgs/pulls)

**Notify maintainers**

@fgaz 
-----

Note for maintainers: Please tag this issue in your PR.

### PR DESCRIPTION
**Pull request by zendo**
###### Description of changes

https://gitlab.gnome.org/World/obfuscate/-/compare/0.0.7...0.0.9

Close: #112324

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/graphics/gnome-obfuscate/default.nix [hunk 1]
```diff
@@ -1,73 +1,67 @@
...
 , fetchFromGitLab
-
 , gettext
 , meson
 , ninja
 , pkg-config
-, python3
 , rustPlatform
 , wrapGAppsHook4
-
```
Comment (fgaz) [inline pkgs/applications/graphics/gnome-obfuscate/default.nix:12]: Could you leave the empty lines? I sometimes use them to separate buildinputs/nativebuildinputs/other

Comment (fgaz) [reply pkgs/applications/graphics/gnome-obfuscate/default.nix:12]: Nevermind, I see you moved some inputs to their correct place, no need to shuffle arguments around then

File: pkgs/applications/graphics/gnome-obfuscate/default.nix [hunk 2]
```diff
@@ -1,73 +1,67 @@
...
 
-  postPatch = ''
-    patchShebangs build-aux/meson_post_install.py
-  '';
-
   meta = with lib; {
     description = "Censor private information";
     homepage = "https://gitlab.gnome.org/World/obfuscate";
     license = licenses.gpl3Plus;
+    platforms = platforms.unix;
```
Comment (fgaz) [inline pkgs/applications/graphics/gnome-obfuscate/default.nix:63]: Why unix?

Comment (zendo) [reply pkgs/applications/graphics/gnome-obfuscate/default.nix]: Anything is OK.

**Comment by vdot0x23** [comment]
- [x] package build on x86_64-linux
- [x] executables tested on x86_64-linux

**Pull request merged**